postgres/macros.jinja

```diff
@@ -4,7 +4,7 @@
 
   {%- filter indent(4) %}
     {%- for k, v in kwarg|dictsort() %}
-- {{ k }}: {{ v }}
+- {{ k }}: {{ "'" if v is string else '' }}{{ v }}{{ "'" if v is string else '' }}
```

Comment:
@sticky-note Perhaps this can be done more cleanly using https://docs.saltstack.com/en/latest/topics/jinja/#quote.  Not tried this yet but how about:

```suggestion
- {{ k }}: {{ v|quote if v is string else v }}
```

Reply:
`| quote `  doesn't work with `:` in strings for example.
`yaml_dquote` has better escaping power and encapsulate into `"` like I wanted